ecosystem/sf-indexer/aptos-sf-indexer/src/substream_processors/block_output_processor.rs

```diff
@@ -455,10 +457,15 @@ impl SubstreamProcessor for BlockOutputSubstreamProcessor {
         );
 
         match tx_result {
-            Ok(_) => Ok(ProcessingResult::new(
-                self.substream_module_name(),
-                block_height,
-            )),
+            Ok(_) => {
+                LATEST_PROCESSED_VERSION
+                    .with_label_values(&[self.substream_module_name()])
+                    .set(last_version);
+                Ok(ProcessingResult::new(
+                    self.substream_module_name(),
+                    block_height,
+                ))
+            }
```

Comment:
Can we have some kind of prom for error below?

Reply:
We have it already in transaction_processors.rs. The event is PROCESSOR_ERRORS.